app/src/controllers/user/selectors.js

```diff
@@ -58,3 +58,5 @@ export const availableProjectsSelector = createSelector(
 );
 
 export const apiKeysSelector = (state) => userSelector(state).apiKeys || [];
+
+export const activeProjectKeySelector = (state) => userSelector(state).activeProjectKey;
```

Comment:
Is it a replacement for `activeProjectSelector`?

Reply:
Done.